actor/builtin/miner/miner.go

```diff
@@ -33,12 +33,12 @@ func init() {
 // largest sector size - this constant and consensus.AncestorRoundsNeeded will
 // need to be reconsidered.
 // https://github.com/filecoin-project/specs/pull/318
-const LargestSectorSizeProvingPeriodBlocks = 1000
+const LargestSectorSizeProvingPeriodBlocks = 300
```

Comment:
why not make even smaller? (50, 100)

Reply:
A single powerful core takes ~40 minutes (80 blocks) to compute a PoSt, so these values give about double that time, leaving room for various delays, reorgs etc. Multi-core machines can compute the PoSt much faster, but not assuming that seems good for now so we don't surprise miners too much.